plugins/modules/net_tools/nmcli.py

```diff
@@ -90,11 +90,52 @@
         version_added: 3.2.0
     routes4:
         description:
-            - The list of ipv4 routes.
-            - Use the format '192.0.3.0/24 192.0.2.1'
+            - The list of IPv4 routes.
+            - Use the format C(192.0.3.0/24 192.0.2.1).
```

Comment:
How about:
```suggestion
            - Use the format C(192.0.3.0/24 192.0.2.1).
            - To specify more complex routes, use the I(routes4_extended) option.
```
(and something similar for `routes6`?)